frontend/lib/pages/food_item_page.dart

```diff
@@ -11,6 +17,31 @@ class FoodItemPage extends StatefulWidget {
 }
 
 class _FoodItemPageState extends State<FoodItemPage> {
+  final nameInput = TextEditingController();
+  final weightInput = TextEditingController();
+  final description = TextEditingController();
+  final category = TextEditingController();
+  final serving_size = TextEditingController();
+  String dropdownValue = 'Vegetables';
+
+  final api = 'http://127.0.0.1:8000/add_food';
```

Comment:
I think this should be located in a separate class since I would prefer if the api link would not be located in the page files, also later on we would use a config file to keep the url link to our backend where this link changes when it's development/production.

Reply:
Yeah I agree! I fixed that as I was working on the get request